server/src/domain/person/person.service.ts

```diff
@@ -78,11 +80,19 @@ export class PersonService {
 
   async getAll(auth: AuthDto, dto: PersonSearchDto): Promise<PeopleResponseDto> {
     const { machineLearning } = await this.configCore.getConfig();
-    const people = await this.repository.getAllForUser(auth.user.id, {
+    const userIds: string[] = [auth.user.id];
+    if (dto.withPartners) {
+      const partners = await this.partnerRepository.getAll(auth.user.id);
+      const partnersIds = partners
+        .filter((partner) => partner.sharedBy && partner.sharedWith && partner.sharedById != auth.user.id) //TODO should && partner.inTimeline be included, as in assets?
```

Comment:
I think I would personally argue yes, however that's certainly debatable...

Reply:
Thanks for the review! 

I'm totally fine with both! Leaving the decision to someone with more milage on the project :)

Reply:
kind of landed on the query parameter being sufficient, if you don't mind :)  

I think that, from a user expectation view, you're communicating your intent ("Also give me my partners photos/faces") pretty explicit if you add the query parameter.  requiring some magic setting in the database too would be "surprising behaviour".